README.md

```diff
@@ -103,6 +103,12 @@ Install the following tools, if you don't have them already:
 > which affects how you clone it. 
 > The GitHub documentation has general help on [forking][] and [cloning][] repos.
 
+If you are working in the Windows desktop environment, make sure you have Git
+`autocrlf` set to `input` **before** initialising the submodules. This will
+ensure that line endings are handled correctly _before_ any Bash scripts are
+executed within the submodules.
+See flutter/website#6201 for details.
```

Comment:
This should be a full link. Or maybe a link to better documentation.

Reply:
sure thing.. I had thought it best to lean on the auto-linking functionality provided by GitHub, but I've got no issue with making it a full link.

Reply:
@domesticmouse, just to confirm: since I've just pushed a new commit with the requested change, should **I** mark this conversation as resolved, or leave it as-is for the moment?